torchvision/transforms/transforms.py

```diff
@@ -905,15 +906,13 @@ def _check_input(self, value, name, center=1, bound=(0, float('inf')), clip_firs
         # if value is 0 or (1., 1.) for brightness/contrast/saturation
         # or (0., 0.) for hue, do nothing
         if value[0] == value[1] == center:
-            value = None
-        return value
+            return torch.FloatTensor()
+        return torch.FloatTensor(value)
```

Comment:
This can probably be kept as a `float`, by annotating it with `Optional[float]`.

Then, in the `forward` where it's used, you'll need to do something like
```python
if fn_id == 0 and self.brightness is not None:
    brightness = self.brightness
    assert brightness is not None
    ...
```
this is needed to tell torchscript that it can specialize the type of the argument to float, instead of `Optional[float]`.

Reply:
`_check_input` returns a list of 2 items, not a float. I could change the naming of `value` because it's misleading. 

Since it's returning a list, I think a tensor is the only way to go?

Reply:
You can also return an `Optional[Tuple[float, float]]` if it's always two elements that are returned, or `Optional[List[float]]` if you really want a list.

Reply:
I found out that just converting the input to float in `_check_input` makes it torchscript happy. The tests pass on Jarvis (at least the test_functional_tensor.py: https://travis-ci.org/github/pytorch/vision/jobs/696793453 )

I didn't need the type nor the assert. If you want them I'm happy to add them, but I thought I'd let you know they don't seem required.

I've also added a simple test with an int as input for the torchscript.